lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -564,7 +563,9 @@ export default class BitcoinProcessor {
           transactionNumber: TransactionNumber.construct(transactionBlock, transactionIndex),
           transactionTime: transactionBlock,
           transactionTimeHash: transactionHash,
-          anchorString: data.slice(this.sidetreePrefix.length)
+          anchorString: data.slice(this.sidetreePrefix.length),
+          feePaid: 100, // issue #216: read actual data from tx
```

Comment:
I don't think #216 will give the fee paid, but we can create another issue to check this in separately if you want.